ext/gd/libgd/gd_bmp.c

```diff
@@ -68,8 +70,10 @@ void * gdImageBmpPtr(gdImagePtr im, int *size, int compression)
 	void *rv;
```

Comment:
nit: I guess you can initialise it here saving 2 lines but that s nothing.

Reply:
Yeah, but I like to stay as close to upstream as possible (I still haven't completely lost hope that at some point in time we can unbundle libgd). And there we have https://github.com/libgd/libgd/blob/77adfd27f776352e8095d98ef4902c28975d19af/src/gd_bmp.c#L88-L99